test-one/src/index.js

```diff
@@ -1,14 +1,23 @@
 import { log } from "./helpers/logger";
 
-export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
-
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+export const BossHog = (num) => {
+  if (num % 3 === 0 && num % 5 === 0) {
```

Comment:
can you think of a simpler way to write this evaluation?

Reply:
Another way could be using the switch statement like below:

  switch (true) {
    case num % 15 === 0:
      return 'BossHog';
    case num % 5 === 0:
      return 'Hog';
    case num % 3 === 0:
      return 'Boss';
    default:
      return num;
  }

Benefits of using switch statement:
1. Better readability: It would be easier to read and understand the code.
2. Better performance: It would be much faster if we are select from a large group of values.

In addition to that, I can also refactor the expression (num % 15 === 0) as a function 'modulus' and pass the function into each case like below.

const modulus = (num, divisor) => {
  return num % divisor === 0;
};

export const BossHog = (num) => {
  switch (true) {
    case modulus(num, 15):
      return 'BossHog';
    case modulus(num, 5):
      return 'Hog';
    case modulus(num, 3):
      return 'Boss';
    default:
      return num;
  }
};

Reply:
Sorry I wasn't able to make the code looks nicer on the comment. But I do use 'Prettier' with VS code and that's how I keep my code format consistent :)